backstop_data/engine_scripts/puppet/contact/find-and-merge-edit-rule-household.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
```

Comment:
since all this files have the `find-and-merge` prefix, wouldn't it make sense to have a `contact/find-and-merge/` folder? what do you think

Reply:
I think thats a nice idea! Let me do it.